hw2/05-form.js

```diff
@@ -1 +1,44 @@
-// Add your code here
+const labelStyle = document.querySelectorAll("label");
+const form = document.querySelector("form");
+const checkBoxStyle = document.querySelectorAll("span");
+const inputStyle = document.querySelectorAll("input");
+
+function handleSubmitForm(event) {
+    event.preventDefault();
+
+    const name = document.getElementById("userName").value;
+    const email = document.getElementById("userEmail").value;
+    const regStatus = document.getElementById("regStatus").value;
+    const coursesTaken = [];
+    const checkedBoxes = document.getElementsByName("coursesTaken");
+    const addComment = document.getElementById("comment").value;
+    // const comment = addComment.value;
+
+    checkedBoxes.forEach(function (checkedBox) {
+        if (checkedBox.checked) {
+            coursesTaken.push(checkedBox.value);
+        }
+    });
+
+    console.log(`Name: ${name}`);
+    console.log(`Email: ${email}`);
+    console.log(`Registration Status: ${regStatus}`);
+    console.log(`Courses taken: ${coursesTaken.join(", ")}`);
+    console.log(`Comments: ${addComment}`);
+}
+
+// html styling
+labelStyle.forEach(function (label) {
+    label.style.fontWeight = "bold";
```

Comment:
Also, it is better to use CSS instead of js for styling.

Reply:
its an option.  Didnt want to add an extra file.  This styling was specific to this form.

Reply:
Understood. What I meant was to add bootstrap classes. There are classes to make font-weight bold, thin, underlined etc.

Reply:
I see.  I thought it would be simpler to just add couple lines of js style and apply it to all label tags.  Instead of adding the bootstrap styling to every single label tag.